lib/vsc/utils/missing.py

```diff
@@ -332,6 +333,46 @@ def get_subclasses(klass, include_base_class=False):
     return get_subclasses_dict(klass, include_base_class=include_base_class).keys()
 
 
+def _extract_module_names(filelist):
+    """
+    Given a list of files, strip off the suffix ('.py') and hide _prefix
+    files except for __init__.py
+    """
+    module_regexp = re.compile(r"^(?P<modname>[^_].*|__init__)\.py$")
+    return [res.group('modname') for res in map(module_regexp.match, filelist) if res]
+
+
+def _trim_leading_sep(path):
+    """
+    If the first char is a path separator (/), return the path with the
+    separator trimmed off.
+    """
+    if path.startswith(os.sep):
+        return path[1:]
+    return path
+
+def _find_zipped_modules(zip_path, pkg_path):
+    """
+    Given a path to a zipped egg, and a pkg path, return the submodules inside
+    the module
+    """
+    zf = zipfile.ZipFile(zip_path)
+    names = zf.namelist()
+    prefix = os.path.commonprefix([zip_path, pkg_path])
+    zip_pkg_path = _trim_leading_sep(pkg_path[len(prefix):])
+    names = [_trim_leading_sep(n[len(zip_pkg_path):]) for n in names if n.startswith(zip_pkg_path)]
+
+    if names:
```

Comment:
An egg where the module isn't found. e.g. looking for `easybuild/easyconfigs` in `/path/to/pkgs/hod.egg` In this case, `hod.egg` will have a common prefix, but the filter clause of the list comprehension will show that no modules in the egg start with `easybuild/easyconfigs`.